articles/aks/use-managed-identity.md

```diff
@@ -52,6 +52,14 @@ A successful cluster creation using managed identities contains this service pri
   }
 ```
 
+You'll also have information to indicate what type of managed dentities that your cluster is refering, for instance, an AKS cluster with system-assigned managed identity enabled contains the following information:  
```

Comment:
We are planning to remove system vs. user assigned terminology from our AKS MI docs . Could you just remove that part for now?